File: app/src/main/java/mb00/android/codehub/ui/user/inject/UserStarredFragmentComponent.java
package mb00.android.codehub.ui.user.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.user.view.UserStarredFragment;


@Subcomponent(modules = UserStarredFragmentModule.class)
public interface UserStarredFragmentComponent extends AndroidInjector<UserStarredFragment> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<UserStarredFragment> {}

}

File: app/src/main/java/mb00/android/codehub/ui/user/inject/UserStarredFragmentComponent.kt
package mb00.android.codehub.ui.user.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.user.view.UserStarredFragment


@Subcomponent(modules = [
    UserStarredFragmentModule::class
])
interface UserStarredFragmentComponent : AndroidInjector<UserStarredFragment> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<UserStarredFragment>()

}
